code-challenges.test.js

```diff
@@ -28,29 +64,81 @@
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 7, 9, 199]
-
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+    //Psuedo-Code for Jest Test:
+        // A describe method that lists the name of the function OR naming of the particular test.
+        describe("findOdds", () => {
+           // A test/it method, nested within the describe block, that in plain words, describe what the function does. 
+           it("returns an array of only the odd numbers, which is then sorted from least to greatest", () => {
+            const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+            // Expected output: [-9, 7, 9, 199]
+            
+            const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+            // Expected output: [-823, 7, 23]
+              // An expect method, nested within the test block, calling on findOdds function, followed by the toEqual() matcher that checks the expected output of the function return. Create 2 expects because there are two arrays to be passed as arguments. 
+              expect(findOdds(fullArr1)).toEqual( [-9, 7, 9, 199] )
+              expect(findOdds(fullArr2)).toEqual( [-823, 7, 23] )
+           })
+        })
+        
+        
 
 
 // b) Create the function that makes the test pass.
+        //Pseudo-Code:
+            // Input: fullArr1 and fullArr2
+            // Output: Only the odd numbers from least to greatest. fullArr1 = Expected output: [-9, 7, 9, 199] and fullArr2 = Expected output: [-823, 7, 23]
+            // Process:
+                // Create a function called findOdds that takes array as the parameter. 
+                // Declare a variable called oddOnly, take in that array and use .filter to loop through an array. 
+                // Using typeof, only return the value that is not a string, boolean, null. And only return the value that is odd. 
+                // Return the array sorted from least to greatest using the buit in method .sort().
+const findOdds = (array) => {
+let oddOnly =  array.filter(value => {
+        return typeof value !== "string" && typeof value !== "boolean" && typeof value !== null && value % 2 !== 0 
+    })
+    return oddOnly.sort((a,b) => a - b)
+    
+}
```

Comment:
Looks great!  For a stretch, you could simplify this a little more by returning `typeof value === "number" && value % 2 !== 0 .  Also, I appreciate the detail you put into your pseudocode as well!